.github/workflows/pytest.yml

```diff
@@ -0,0 +1,101 @@
+name: Tests
+
+on: [push, pull_request]
+
+jobs:
+  ubuntu:
+
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+        use-conda: [true, false]
+        use-dist: [false]
+        include:
+          - python-version: 3.8
+            code-cov: true
+          - python-version: 3.7
+            use-conda: false
+            use-dist: true
+      fail-fast:  false
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Conda Install test dependencies
+      if: matrix.use-conda == true
+      run: |
+        # Miniconda is available in $CONDA env var
+        $CONDA/bin/conda create -n testenv --yes pip wheel gxx_linux-64 gcc_linux-64 swig python=${{ matrix.python-version }}
+        $CONDA/envs/testenv/bin/python3 -m pip install --upgrade pip
+        $CONDA/envs/testenv/bin/pip3 install -e .[all]
+    - name: Install test dependencies
+      if: matrix.use-conda == false && matrix.use-dist == false
+      run: |
+        python -m pip install --upgrade pip
+        if [[ `python -c 'import platform; print(platform.python_version())' | cut -d '.' -f 2` -eq 6 ]]; then
+          # Numpy 1.20 dropped suppert for Python3.6
+          pip install "numpy<=1.19"
+        fi
+        pip install -e .[all]
+        sudo apt-get update
+        sudo apt-get remove swig
+        sudo apt-get install swig3.0
+        sudo ln -s /usr/bin/swig3.0 /usr/bin/swig
+    - name: Dist Install test dependencies
+      if: matrix.use-conda == false && matrix.use-dist == true
+      run: |
+        python -m pip install --upgrade pip
+        sudo apt-get update
+        sudo apt-get remove swig
+        sudo apt-get install swig3.0
+        sudo ln -s /usr/bin/swig3.0 /usr/bin/swig
+        # We need to install for the dependencies, like pytest
+        python setup.py sdist
+        last_dist=$(ls -t dist/smac-*.tar.gz | head -n 1)
+        pip install $last_dist[all]
+    - name: Store repository status
+      id: status-before
+      run: |
+        echo "::set-output name=BEFORE::$(git status --porcelain -b)"
+    - name: Conda Run tests
+      timeout-minutes: 45
+      if: matrix.use-conda == true
+      run: |
+        export OPENBLAS_NUM_THREADS=1
+        export OMP_NUM_THREADS=1
+        export MKL_NUM_THREADS=1
+        # We activate conda as metalearning uses python directly, so we need
```

Comment:
This looks like a copy-paste which is not needed here.

Reply:
Sorry, removed.